tests/conformance/def.bzl

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 The Kubernetes Authors.
```

Comment:
lic should be changed ?

Reply:
From what I saw if its a new file we write the current year

Reply:
question i about `# Copyright 2021 The Kubernetes Authors.` 
everywhere else its Kubevirt Authors

Reply:
Oh totally missed that :man_facepalming:, done.